README.md

```diff
@@ -1 +1,149 @@
-# Metrics-webapp
\ No newline at end of file
+# # Metrics-webapp
+[![Netlify Status](https://api.netlify.com/api/v1/badges/aa43f99a-9807-4c35-931a-5dbaf90bd75a/deploy-status)](https://app.netlify.com/sites/riyana5678/deploys)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<!-- TABLE OF CONTENTS -->
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_spanner: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:rocket: Live Demo](#live-demo)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [:silhouettes: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:question: FAQ](#faq)
+- [:memo: License](#license)
+<!-- PROJECT DESCRIPTION -->
+
+📖 [Metrics-webapp]
+In this project, I set up my Metrics WebApp project. It is a mobile web application to check a list of metrics about world weather by choosing a country one can know the current weather of such country.
+
+# 🛠 Built With
+
+HTML
+
+CSS
+
+Javascript
+
+React JS
+
+Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+
+Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+ESlint (is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs).
+🛠 Built With Tech Stack Client Server Database
+
+# 🚀 Live Demo
+Please use the following link to access the website:
+[live demo](https://riyana5678.netlify.app/)
+[live presentation](https://www.loom.com/share/fddfc6960af443308443f38ba92144b6)
```

Comment:
[OPTIONAL] Kindly give your live presentation a heading and put it on a separate line. I almost made the absence of a video link a required change